CHANGELOG.adoc

```diff
@@ -25,6 +25,11 @@ Updated Motoko from 0.6.11 to 0.6.14.
 * Improve performance of bignum equality
 * Stable signatures: frontend, metadata, command-line args
 
+== Cycles wallet
+
+Module hash: 983acc59e65d470e4e3ba97512864cdc9a2b560caaca1c89afc9f2fc78aa2643
+https://github.com/dfinity/cycles-wallet/commit/0f4e6c9f243f2a19caef7f7c552c9adc6ad43140
```

Comment:
Now that we have a new commit in the cycles-wallet repo main branch (57e53fcb679d1ea33cc713d2c0c24fc5848a9759), let's regenerate the wallet.wasm from that and update these hashes.